### DISCUSSION TITLE
Is the Wallet Section within the Rainbow Kit Swipeable?

### DISCUSSION THREAD

**Discussion by LouisJung1** [rainbow-me/rainbowkit#1071] [Q&A]
Is the wallet section within the rainbow kit in the attached picture swipeable? 
The wallet that is currently being displayed on PC is not visible on mobile.
![스크린샷 2023-03-10 오후 2 43 03](https://user-images.githubusercontent.com/124356253/224236150-510fea92-911e-4929-8755-3b6341364ed0.png)

**Comment by DanielSinclair** [comment]
@LouisJung1 Yes, this is swipeable. Some wallets will only support desktop or mobile and these are hidden according to the platform. What wallet isn't appearing?
